ipn/store.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ipn
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"os"
+	"sync"
+
+	"tailscale.com/atomicfile"
+)
+
+// ErrStateNotExist is returned by StateStore.ReadState when the
+// requested state id doesn't exist.
```

Comment:
Go style is to capitalize ID. (mentioned at https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Sorry?

Reply:
Fixed in this file, although I would point out that the entire rest of the codebase doesn't follow this. I'm fine with either, but done consistently. I wonder if there's a clean way to machine-rewrite...